index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+<body>
+  
+  <header class="header">
+    <img src="images/trybewarts-header-logo.svg" alt="" class="trybewarts-header-logo">
+    <h1 id="trybewarts-header-title" >Trybewarts</h1>
+    <form action="" class="trybewarts-login">
```

Comment:
Amigos, no caso do uso da tag form, vamos adotar usar o atributo actions, pelo menos com o link local '#', veja a sugestão:
```suggestion
    <form action="#" class="trybewarts-login">
```